levels/castle_inside/clock_hour_hand/geo.inc.c

```diff
@@ -1,6 +1,6 @@
 // 0x0E001548
 const GeoLayout castle_geo_001548[] = {
-   GEO_CULLING_RADIUS(300),
+   GEO_CULLING_RADIUS(500),
```

Comment:
Is this change still necessary?

Reply:
Yes, without it, the clock hands get culled at the very edge of the BitDW hallway (and most likely the other end of the castle courtyard)